src/api/ledger/orderbook.js

```diff
@@ -77,7 +78,7 @@ function getOrderbookAsync(account, orderbook, options, callback) {
 }
 
 function getOrderbook(account: string, orderbook: Object, options={}) {
-  return utils.promisify(getOrderbookAsync.bind(this))(
+  return utils.promisify(getOrderbookAsync).call(this,
```

Comment:
basically `const ref = utils.promisify(getOrderbookAsync);` or whatever form makes sense

not a big deal, just throwing it out there